### PR TITLE
Adding docs for snapshot command

### PR DESCRIPTION
**Pull request by fhnaseer**
Adding docs for snapshot command

### CODE REVIEW

File: docs/core/additional-tools/dotnet-coverage.md [hunk 1]
```diff
@@ -45,11 +45,12 @@ The `dotnet-coverage` tool:
 |-----------------------------------------------------------|
 | [dotnet-coverage collect](#dotnet-coverage-collect)       |
 | [dotnet-coverage merge](#dotnet-coverage-merge)           |
+| [dotnet-coverage snapshot](#dotnet-coverage-snapshot)     |
 | [dotnet-coverage shutdown](#dotnet-coverage-shutdown)     |
 
 ## dotnet-coverage collect
 
-The `collect` command is used to collect code coverage data for any .NET process and its subprocesses. For example, you can collect code coverage data for a console application or a Blazor application. This command is available on Windows (x86 and x64) and Linux (x64). The command supports only .NET modules. Native modules are not supported.
+The `collect` command is used to collect code coverage data for any .NET process and its subprocesses. For example, you can collect code coverage data for a console application or a Blazor application. This command is available on Windows (x86 and x64), Linux (x64), and macOS (x64). The command supports only .NET modules. Native modules are not supported.
```
Comment (jakubch1) [inline docs/core/additional-tools/dotnet-coverage.md:53]: remove , before and

**Comment by jakubch1** [comment]
@tdykstra could you please review this PR?

File: docs/core/additional-tools/dotnet-coverage.md (hunk 1 above)
Comment (Youssef1313) [reply docs/core/additional-tools/dotnet-coverage.md:53]: @jakubch1 I think the comma should remain. But @gewarren can confirm.

Comment (BillWagner) [reply docs/core/additional-tools/dotnet-coverage.md:53]: The Oxford comma is the approved style. This should stay.

**Comment by BillWagner** [comment]
@fhnaseer @jakubch1 Tom is on vacation until the first week of January. I gave it a read, and I didn't find any issues. 

Tom will review and give final approval when he returns from break.

Thanks for your patience.

File: docs/core/additional-tools/dotnet-coverage.md [hunk 2]
```diff
@@ -193,6 +229,12 @@ Session ID was specified as `serverdemo`. Then you can run tests as follows:
 D:\serverexample\tests> dotnet test
 ```
 
+Code coverage file for session `serverdemo` can be generated with current coverage as follows:
+
+```console
+dotnet-coverage snapshot --output after_first_test.coverage serverdemo.
```
Comment (Youssef1313) [inline docs/core/additional-tools/dotnet-coverage.md:235]: Is the trailing period intended?

File: docs/core/additional-tools/dotnet-coverage.md [hunk 3]
```diff
@@ -141,6 +142,41 @@ dotnet-coverage merge [-?|-h|--help] [-l|--log-file <log-file>] [-ll|--log-level
...
+```console
+dotnet-coverage snapshot [-?|-h|--help] [-l|--log-file <log-file>] [-ll|--log-level <log-level>]
+  [-o|--output <output>] [-r|--reset] <session>
+```
+
+### Arguments
+
+- **`<session>`**
+
+  The session ID of the collection for which coverage file is to be generated.
```
Comment (tdykstra) [inline docs/core/additional-tools/dotnet-coverage.md:160]:
```suggestion
  The session ID of the collection for which a coverage file is to be generated.
```

File: docs/core/additional-tools/dotnet-coverage.md [hunk 4]
```diff
@@ -141,6 +142,41 @@ dotnet-coverage merge [-?|-h|--help] [-l|--log-file <log-file>] [-ll|--log-level
...
+
+  Sets the log file path. When you provide a directory (with a path separator at the end), a new log file is generated for each process under analysis.
+
+- **`-ll|--log-level <log-level>`**
+
+  Sets the log level. Supported values: `Error`, `Info`, and  `Verbose`.
+
+- **`-o|--output <output>`**
+
+  Sets the code coverage report output file. If not provided then it is generated automatically with timestamp.
```
Comment (tdykstra) [inline docs/core/additional-tools/dotnet-coverage.md:174]:
```suggestion
  Sets the code coverage report output file. If not provided, it's generated automatically with a timestamp.
```

File: docs/core/additional-tools/dotnet-coverage.md [hunk 5]
```diff
@@ -141,6 +142,41 @@ dotnet-coverage merge [-?|-h|--help] [-l|--log-file <log-file>] [-ll|--log-level
...
+
+- **`-r|--reset <reset>`**
+
+  If provided then existing coverage information is cleared after coverage file is created.
```
Comment (tdykstra) [inline docs/core/additional-tools/dotnet-coverage.md:178]:
```suggestion
  Clears existing coverage information after a coverage file is created.
```

File: docs/core/additional-tools/dotnet-coverage.md [hunk 6]
```diff
@@ -193,6 +229,12 @@ Session ID was specified as `serverdemo`. Then you can run tests as follows:
 D:\serverexample\tests> dotnet test
 ```
 
+Code coverage file for session `serverdemo` can be generated with current coverage as follows:
```
Comment (tdykstra) [inline docs/core/additional-tools/dotnet-coverage.md:232]:
```suggestion
A code coverage file for session `serverdemo` can be generated with current coverage as follows:
```

**Review by tdykstra — COMMENTED** [review]
Looks great, I left just a few nit suggestions. Ready to merge when those and the comment by @Youssef1313 are addressed.

File: docs/core/additional-tools/dotnet-coverage.md (hunk 2 above)
Comment (fhnaseer) [reply docs/core/additional-tools/dotnet-coverage.md]: removed

**Review by tdykstra — APPROVED** [review]

**Pull request merged**